style.css

```diff
@@ -0,0 +1,361 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@media only screen and (max-width: 375px) {
```

Comment:
- [ ] [Optional] Kindly increase the max-width to 768px to get all the mobile devices.